http/database_example.http

```diff
@@ -0,0 +1,18 @@
+@hostname=localhost
+@port=4444
+@address=http://{{hostname}}:{{port}}
+
+### Get document by ID
+
+GET {{address}}/read/1
```

Comment:
If that one is an ID please move it be a variable so that we know what is required and what is a variable